### PR TITLE
Small refactor & update in modlist.menu, server_browser.menu

### PR DESCRIPTION
**Pull request by eve-v0**
Update modlist.menu
Refactor
Doesnt change visuals

Update server_browser.menu
Makes text in "Join" button allign to middle/north which looks better
image
image

### CODE REVIEW

**Review by ASpoonPlaysGames — APPROVED** [review]
Code looks good (there isn't really any code per-say but yeah the diff looks good)

**Review by Zanieon — APPROVED** [review]
Good finding.

File: Northstar.Client/mod/resource/ui/menus/modlist.menu [hunk 1]
```diff
@@ -407,11 +407,11 @@ resource/ui/menus/modlist.menu
 			
 			labelText	"#CLEAR_FILTERS"
 			wide		100
-			xpos		-15
-			ypos		-55
+			xpos		0
+			ypos		0
```
Comment (Alystrasz) [inline Northstar.Client/mod/resource/ui/menus/modlist.menu:411]: I don't see any difference regarding vertical centering here?

Comment (eve-v0) [reply Northstar.Client/mod/resource/ui/menus/modlist.menu:411]: centering change is only for the Join button in server_browser.menu

Comment (ASpoonPlaysGames) [reply Northstar.Client/mod/resource/ui/menus/modlist.menu:411]: Where was vertical centring mentioned? Based on the PR description this PR should shift the text a bit to the right to centre it properly

Comment (eve-v0) [reply Northstar.Client/mod/resource/ui/menus/modlist.menu:411]: idk i didint mention vertical centering

Comment (Alystrasz) [reply Northstar.Client/mod/resource/ui/menus/modlist.menu:411]: `ypos` was modified so I assumed vertical position was changed as well

Comment (eve-v0) [reply Northstar.Client/mod/resource/ui/menus/modlist.menu:411]: horizontal position wasnt changed

Comment (ASpoonPlaysGames) [reply Northstar.Client/mod/resource/ui/menus/modlist.menu:411]: It was replaced by the textAlignment I think

Comment (eve-v0) [reply Northstar.Client/mod/resource/ui/menus/modlist.menu:411]: not replaced

Comment (eve-v0) [reply Northstar.Client/mod/resource/ui/menus/modlist.menu:411]: change in modlist.menu doesnt have anything to do with centering the join button

File: Northstar.Client/mod/resource/ui/menus/modlist.menu [hunk 2]
```diff
@@ -407,11 +407,11 @@ resource/ui/menus/modlist.menu
 			
 			labelText	"#CLEAR_FILTERS"
 			wide		100
-			xpos		-15
-			ypos		-55
+			xpos		0
```
Comment (Alystrasz) [inline Northstar.Client/mod/resource/ui/menus/modlist.menu:410]: What does this change for the mod list button?
I don't see any change with and without the PR:

##### Without

ui-mod-main

##### With

ui-mod-pr

File: Northstar.Client/mod/resource/ui/menus/server_browser.menu [hunk 3]
```diff
@@ -2554,6 +2554,7 @@ resource/ui/menus/mods_browse.menu
 			xpos -17
 			ypos -57
 			zpos 90
+			textAlignment	north
```
Comment (Alystrasz) [inline Northstar.Client/mod/resource/ui/menus/server_browser.menu:2557]: I wanted to suggest `center` for this parameter, but the resulting position is the same as `north` it seems ¯\\\_(ツ)\_/¯

**Pull request closed without merging**